src/Sarif/EnumeratedArtifact.cs

```diff
@@ -160,15 +163,15 @@ private void TryRewindStream()
                 }
                 else if (this.Stream != null)
                 {
-                    this.SizeInBytes = (long)this.Stream.Length;
+                    this.sizeInBytes = (long)this.Stream.Length;
                 }
                 else if (Uri != null && Uri.IsAbsoluteUri && Uri.IsFile)
                 {
-                    this.sizeInBytes = (long)FileSystem.FileInfoLength(Uri.LocalPath);
```

Comment:
>LocalPath [](http://example.com/codeflow?start=75&length=9)

are you sure this change is required now that you've removed the call in ordered file specifier to remove the call to `GetFile`?

It doesn't make sense to me that `LocalPath` would be broken unless something that intervenes between the file enumeration and this code path has done something unexpected to the URI.

Thinking it all over, I now believe the root cause is for relative file paths that work against the file system. Or at least I'd be very sure to look at this case.